### DISCUSSION TITLE
If a URL was passed, is the URL requested or a domain?

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [ddosify/ddosify#62] [Q&A] [answered]
Sorry, I can not check this by myself because I am a developer in other languages. I do not develop in Go.
If a URL was passed, is the URL requested or a domain?

**Comment by kursataktas** [comment, marked as answer]
Hi,

If you are asking the `-t` flag in CLI, or the `URL` field in the config file, you should pass the full endpoint URL that you want to send requests.

If not can you give me an example about the issue?

**Reply by simon-liebehenschel** [reply]
I am asking about the CLI.

I think that I already understand that if a URL was passed, then the passed URL is requested (the URL is not cut off to domain only).

**Reply by kursataktas** [reply]
Yes, `-t` flag is for target endpoint, all of the below are valid;

```
https://target.com
https://target.com/api/endpoint
https://target.com/api/endpoint?query=searchterm
```
